### DISCUSSION TITLE
Apache2 Default Page After Restart ?

### DISCUSSION THREAD

**Discussion by SaasX** [postalserver/postal#2302] [Help with using Postal]
Hello,

After a restart of the server, I come across the "Apache2 Default Page".
(Everything was working fine before the reboot.)

Postal starts well and works (status check).
I don't understand where this problem comes from, a service to restart in addition to postal?

**Comment by willpower232** [comment]
You will need to check your apache virtualhost configuration to make sure that it is still correct, I'm not sure what that would look like off the top of my head but feel free to share it here if you need help spotting what is wrong

**Reply by SaasX** [reply]
The configuration redirects to /var/www/html since the restart.
The redirection was done without my intervention during the installation.
In which folder is postal installed?
Thank you

**Reply by willpower232** [reply]
Postal adds some files to /opt/postal but mostly runs as Docker containers in version 2. You would have needed to set up some proxy to its ports during installation.

**Reply by SaasX** [reply]
Hello,

I use caddy during installation :

docker run -d \
   --name postal-caddy \
   --restart always \
   --network host \
   -v /opt/postal/config/Caddyfile:/etc/caddy/Caddyfile \
   -v /opt/postal/caddy-data:/data \
   caddy

All work fine but if i restart the server,  don't work after.

**Reply by willpower232** [reply]
ah right so you have apache installed on your server but you were intending to use caddy from your docker setup.

You will need to do something like `sudo systemctl disable apache2` and then restart again and apache should not come back. You may be able to remove it but that would depend on what installed it in the first place.
